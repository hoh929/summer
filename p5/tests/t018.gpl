// when a double is passed to random, it should be rounded down
// On some operations systems (especially MAC) the random values may be different than the posted values
//
// *** MAKE SURE ALL the values your program generates are random and 0 <= < 10 ***
//
int i0 = random(10.123);
int i1 = random(10.123);
int i2 = random(10.123);
int i3 = random(10.123);
int i4 = random(10.123);
int i5 = random(10.123);
int i6 = random(10.123);
int i7 = random(10.123);
int i9 = random(10.123);
double j0 = random(10.123);
double j1 = random(10.123);
double j2 = random(10.123);
double j3 = random(10.123);
double j4 = random(10.123);
double j5 = random(10.123);
double j6 = random(10.123);
double j7 = random(10.123);
double j9 = random(10.123);
