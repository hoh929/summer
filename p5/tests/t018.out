gpl.cpp::main()
  input file(tests/t018.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
int i0 = 6
int i1 = 0
int i2 = 1
int i3 = 1
int i4 = 2
int i5 = 8
int i6 = 1
int i7 = 0
int i9 = 5
double j0 = 3
double j1 = 4
double j2 = 3
double j3 = 7
double j4 = 4
double j5 = 6
double j6 = 2
double j7 = 2
double j9 = 8
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
